lib/crash-reporter.js

```diff
@@ -47,7 +47,11 @@ CrashReporter.on = function() {
     });
 };
 
-CrashReporter.submit = function(report) {
+CrashReporter.submit = function(report, opts) {
+  if (opts === undefined) opts = {}
+
+  const silent = (opts.silent || false)
+
```

Comment:
nits: curly braces for the if-statement's consequent and semi-colons (`grunt` would've caught this :P)